app/scripts/lib/auth-errors.js

```diff
@@ -190,10 +190,6 @@ function (_, Errors) {
       errno: 1021,
       message: UNEXPECTED_ERROR_MESSAGE
     },
-    ILLEGAL_IFRAME_PARENT: {
-      errno: 1022,
-      message: t('Firefox Accounts can only be placed into an IFRAME on approved sites')
-    },
```

Comment:
Is this the only place this now-obsolete errno was documented?  If may be worth leaving a comment to explain the gap in errno numbering.